extensions/amp-accordion/0.1/test/test-amp-accordion.js

```diff
@@ -845,6 +846,34 @@ describes.realWin(
         content2.getAttribute('aria-labelledby')
       );
     });
+
+    it('should throw an error when amp-bind used with expanded attribute ', async () => {
+      const errors = [];
+      const consoleError = console.error;
+      console.error = (msg) => {
+        errors.push(msg);
+      };
+
+      const html = htmlFor(win.document);
+      const accordion = html`
+        <amp-accordion animate>
+          <section [expanded]="section1">
+            <h2>Section 1</h2>
+            <div>Puppies are cute.</div>
+          </section>
+        </amp-accordion>
+      `;
+      doc.body.appendChild(accordion);
+
+      try {
+        await accordion.buildInternal();
+      } catch (expected) {
+        expect(errors.length).to.equal(1);
+        expect(errors[0]).to.include('The "expanded" attribute');
+      }
```

Comment:
If `buildInternal` runs without error, this test won't fail. We should do:
```
expect(() => accordion.buildInternal()).to.throw(...)
```

Reply:
I did something similar to your suggestion.  Since `buildInternal` is async I need to modify it a bit.  The latest revision will fail if `buildInternal` runs without error.